gio-sys/src/lib.rs

```diff
@@ -13,6 +13,10 @@ extern crate glib_sys as glib;
 extern crate gobject_sys as gobject;
 extern crate libc;
 
+mod manual;
+
+pub use manual::*;
```

Comment:
I'm pretty sure this file should never be edited (as the header says). So that means that we need to generate this file differently, for example generating a `lib.rs` if not present with an include to `auto.rs` and reexport everything from it.

Reply:
This is exactly the same way how it's already done for `glib-sys`.

Reply:
I didn't add the lines manually, they were added by `make`.
I added the file to the commit because it contained changes which were related to this, but I can remove the file from the PR (but then, it would not build without doing make first).

Reply:
Sorry, I was sure it was all in the same file. Nevermind, I was completely wrong here!